torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2021,6 +2021,35 @@ def sample_inputs_logsumexp(self, device, dtype, requires_grad):
 
     return tuple(samples)
 
+def sample_inputs_like_fns(self, device, dtype, requires_grad, **kwargs):
+    inputs: List[Tuple[Tuple, Dict]] = [
```

Comment:
Meta note: typing is lame -- did something complain this wasn't done? If so we should fix that so typing is never required in this file.